frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx

```diff
@@ -453,18 +459,22 @@ export const SlashCommands = forwardRef<SlashCommandsRef, SlashCommandsProps>(fu
     )
 })
 
-const SlashCommandsPopover = forwardRef<SlashCommandsRef, SlashCommandsProps>(function SlashCommandsPopover(
-    props: SlashCommandsProps,
-    ref
-): JSX.Element | null {
-    return (
-        <Popover
-            overlay={<SlashCommands ref={ref} {...props} mode="slash" />}
-            visible
-            referenceElement={props.decorationNode}
-        />
-    )
-})
+export const SlashCommandsPopover = forwardRef<SlashCommandsRef, SlashCommandsPopoverProps>(
```

Comment:
Something in here isn't working. With one node in the book, if I click add and pick something it replaces the node....